### PR TITLE
Support creating materials+textures in Workers

### PR DESCRIPTION
**Pull request by pmconne**
WorkerGraphicDescriptionContext provides methods for creating materials and textures.
Materials are easy, as they don't directly allocate WebGL resources - we just need to put their parameters into the GraphicDescription.
For textures, we store the information needed to allocate the WebGL texture when resolving the context on the main thread.

To make this work more usefully, I had to fix a long-standing limitation of GraphicBuilder: it previously assumed that all the geometry added to the builder would use the same material and texture, because it had no way to do an ordered comparison of materials and textures. I added those ordered comparisons. Since we can't compare pointer values a la C++, each material and texture gets assigned a private GUID upon construction.

The GraphicDescriptionContext-related APIs grew so I moved them out of the GraphicDescription source files to their own files.

GraphicDescriptionBuilder constructor now requires a full context instead of just the constraints it previously required.

### CODE REVIEW

**Comment by pmconne** [comment]
@ben-polinsky I'm not sure what this docs error is trying to tell me. I didn't touch core-backend in this PR, FWIW.
```
--[ FAILURE: @itwin/core-backend ]--------------------------[ 29.79 seconds ]--

/Volumes/Builds/vsts_work_a/76/itwinjs-core/tools/build/scripts/docs.js:22
  throw err;
  ^

SyntaxError: Expected double-quoted property name in JSON at position 35619845
    at JSON.parse (<anonymous>)
    at parseFile (/Volumes/Builds/vsts_work_a/76/itwinjs-core/tools/build/scripts/utils/validateTags.js:42:29)
    at validateTags (/Volumes/Builds/vsts_work_a/76/itwinjs-core/tools/build/scripts/utils/validateTags.js:34:10)
    at /Volumes/Builds/vsts_work_a/76/itwinjs-core/tools/build/scripts/docs.js:98:21
```

**Comment by ben-polinsky** [comment]
> @ben-polinsky I'm not sure what this docs error is trying to tell me. I didn't touch core-backend in this PR, FWIW. I requeued to see if it's reproducible.

No idea, but it seems okay on the re-run.

**Review by DStradley — APPROVED** [review]

**Review by danieliborra — APPROVED** [review]

**Comment by pmconne** [comment]
@ben-polinsky or @aruniverse please review.

File: core/common/src/RenderMaterial.ts [hunk 1]
```diff
@@ -17,16 +18,24 @@ export abstract class RenderMaterial {
   public readonly key?: string;
   /** Describes how to map an image to a surface to which this material is applied. */
   public readonly textureMapping?: TextureMapping;
+  /** Used for ordered comparisons, e.g. in DisplayParams.compareForMerge */
+  private readonly _guid: GuidString;
 
   // eslint-disable-next-line deprecation/deprecation
-  protected constructor(params: RenderMaterial.Params) {
+  protected constructor(params: Pick<RenderMaterial.Params, "key" | "textureMapping">) {
```
Comment (ben-polinsky) [inline core/common/src/RenderMaterial.ts:25]: If `Params` is deprecated (and going to be removed in 5.0?), does it make sense to `Pick` them?

Comment (pmconne) [reply core/common/src/RenderMaterial.ts:25]: addressed.

**Review by ben-polinsky — APPROVED** [review]

**Review by markschlosseratbentley — APPROVED** [review]

**Comment by pmconne** [comment]
Some flakery going on.
```
2024-07-30T13:59:13.9182330Z   1) Default supplemental rules
2024-07-30T13:59:13.9182520Z        Content modifiers
2024-07-30T13:59:13.9182650Z          bis.Element
2024-07-30T13:59:13.9182750Z            Related properties
2024-07-30T13:59:13.9183180Z              loads `Element -> ExternalSourceAspect.Identifier` property into 'Source Information' group:
2024-07-30T13:59:13.9185970Z      Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Volumes/Builds/vsts_work_a/3/s/full-stack-tests/presentation/lib/backend/DefaultSupplementalRules.test.js)
2024-07-30T13:59:13.9186240Z   
2024-07-30T13:59:13.9186300Z 
2024-07-30T13:59:13.9186380Z   2) Content
2024-07-30T13:59:13.9186530Z        Guid properties
2024-07-30T13:59:13.9186620Z          creates guid fields:
2024-07-30T13:59:13.9186900Z      Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Volumes/Builds/vsts_work_a/3/s/full-stack-tests/presentation/lib/frontend/content/GuidProperties.test.js)
2024-07-30T13:59:13.9187110Z   
2024-07-30T13:59:13.9188670Z 
2024-07-30T13:59:13.9188980Z   3) Content
2024-07-30T13:59:13.9189120Z        Error handling
2024-07-30T13:59:13.9189250Z          waits for frontend timeout when request exceeds the backend timeout time:
2024-07-30T13:59:13.9189580Z      Error: Timeout of 60000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Volumes/Builds/vsts_work_a/3/s/full-stack-tests/presentation/lib/frontend/content/Infrastructure.test.js)
2024-07-30T13:59:13.9189960Z   
2024-07-30T13:59:13.9190010Z 
2024-07-30T13:59:13.9190050Z 
2024-07-30T13:59:13.9190080Z 
2024-07-30T13:59:13.9360850Z Returned error code: 3
2024-07-30T13:59:13.9361350Z "presentation-full-stack-tests" failed to build.
```

```
2024-07-30T14:03:15.5958000Z   GeoServices
2024-07-30T14:03:15.5958170Z     ✔ should not be able to interpret an invalid GeographicCRS (1650ms)
2024-07-30T14:03:15.5958330Z     Interpret to completion an incomplete GeographicCRS
2024-07-30T14:03:15.5958520Z       ✔ should be able to interpret to completion britishNationalGridOld (6489ms)
2024-07-30T14:03:15.5958720Z       ✔ should be able to interpret to completion EWRGCS
2024-07-30T14:03:15.5958840Z node:internal/errors:541
2024-07-30T14:03:15.5958950Z       throw error;
2024-07-30T14:03:15.5959040Z       ^
2024-07-30T14:03:15.5959140Z 
2024-07-30T14:03:15.5959340Z TypeError [ERR_INVALID_ARG_TYPE]: The "code" argument must be of type number. Received type string ('128SIGSEGV')
2024-07-30T14:03:15.5959530Z     at process.set [as exitCode] (node:internal/bootstrap/node:123:9)
2024-07-30T14:03:15.5959810Z     at ChildProcess.<anonymous> (/Volumes/Builds/vsts_work_a/3/s/common/temp/node_modules/.pnpm/foreground-child@2.0.0/node_modules/foreground-child/index.js:63:22)
2024-07-30T14:03:15.5960000Z     at ChildProcess.emit (node:events:519:28)
2024-07-30T14:03:15.5960150Z     at maybeClose (node:internal/child_process:1105:16)
2024-07-30T14:03:15.5960320Z     at ChildProcess._handle.onexit (node:internal/child_process:305:5) {
2024-07-30T14:03:15.5960490Z   code: 'ERR_INVALID_ARG_TYPE'
2024-07-30T14:03:15.5960590Z }
2024-07-30T14:03:15.5960630Z 
2024-07-30T14:03:15.5960720Z Node.js v20.16.0
2024-07-30T14:03:15.5960820Z Returned error code: 1
2024-07-30T14:03:15.5960950Z "@itwin/core-backend" failed to build.
```

```
2024-07-30T13:59:01.6289142Z   5 failing
2024-07-30T13:59:01.6289268Z 
2024-07-30T13:59:01.6289445Z   1) GeoCoord
2024-07-30T13:59:01.6289662Z        should get different results for different datums:
2024-07-30T13:59:01.6289820Z 
2024-07-30T13:59:01.6289989Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6290259Z       + expected - actual
2024-07-30T13:59:01.6290406Z 
2024-07-30T13:59:01.6290591Z       -false
2024-07-30T13:59:01.6290780Z       +true
2024-07-30T13:59:01.6290964Z       
2024-07-30T13:59:01.6291370Z       at Context.<anonymous> (D:\vsts_a\19\s\full-stack-tests\core\src\frontend\standalone\GeoCoord.test.ts:63:56)
2024-07-30T13:59:01.6291564Z 
2024-07-30T13:59:01.6291709Z   2) GeoCoord
2024-07-30T13:59:01.6291931Z        should read repeated requests from the cache:
2024-07-30T13:59:01.6292060Z 
2024-07-30T13:59:01.6292226Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6292462Z       + expected - actual
2024-07-30T13:59:01.6292590Z 
2024-07-30T13:59:01.6292784Z       -false
2024-07-30T13:59:01.6292944Z       +true
2024-07-30T13:59:01.6293134Z       
2024-07-30T13:59:01.6293478Z       at Context.<anonymous> (D:\vsts_a\19\s\full-stack-tests\core\src\frontend\standalone\GeoCoord.test.ts:97:56)
2024-07-30T13:59:01.6293956Z 
2024-07-30T13:59:01.6294109Z   3) GeoCoord
2024-07-30T13:59:01.6294337Z        should get some IModelCoords from the cache and calculate some:
2024-07-30T13:59:01.6294508Z 
2024-07-30T13:59:01.6294678Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6294955Z       + expected - actual
2024-07-30T13:59:01.6295114Z 
2024-07-30T13:59:01.6295294Z       -false
2024-07-30T13:59:01.6295485Z       +true
2024-07-30T13:59:01.6295666Z       
2024-07-30T13:59:01.6296041Z       at Context.<anonymous> (D:\vsts_a\19\s\full-stack-tests\core\src\frontend\standalone\GeoCoord.test.ts:143:49)
2024-07-30T13:59:01.6296459Z 
2024-07-30T13:59:01.6296607Z   4) GeoCoord
2024-07-30T13:59:01.6296831Z        should get some GeoCoords from the cache and calculate some:
2024-07-30T13:59:01.6297005Z 
2024-07-30T13:59:01.6297173Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6297447Z       + expected - actual
2024-07-30T13:59:01.6297589Z 
2024-07-30T13:59:01.6297776Z       -false
2024-07-30T13:59:01.6297965Z       +true
2024-07-30T13:59:01.6298146Z       
2024-07-30T13:59:01.6298515Z       at Context.<anonymous> (D:\vsts_a\19\s\full-stack-tests\core\src\frontend\standalone\GeoCoord.test.ts:163:49)
2024-07-30T13:59:01.6298731Z 
2024-07-30T13:59:01.6298881Z   5) GeoCoord
2024-07-30T13:59:01.6299098Z        should get proper result from Geographic CRS conversion:
2024-07-30T13:59:01.6299259Z 
2024-07-30T13:59:01.6299424Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6299693Z       + expected - actual
2024-07-30T13:59:01.6299840Z 
2024-07-30T13:59:01.6300016Z       -false
2024-07-30T13:59:01.6300203Z       +true
2024-07-30T13:59:01.6300385Z       
2024-07-30T13:59:01.6300754Z       at Context.<anonymous> (D:\vsts_a\19\s\full-stack-tests\core\src\frontend\standalone\GeoCoord.test.ts:196:62)
2024-07-30T13:59:01.6300976Z 
2024-07-30T13:59:01.6301063Z 
2024-07-30T13:59:01.6301177Z 
2024-07-30T13:59:01.6302014Z ##[warning]7 tests skipped in core-full-stack-tests
2024-07-30T13:59:01.6302559Z Returned error code: 1
2024-07-30T13:59:01.6302855Z "core-full-stack-tests" failed to build.
2024-07-30T13:59:01.6303016Z 
2024-07-30T13:59:01.6303163Z   542 passing (17m)
2024-07-30T13:59:01.6303358Z   2 pending
2024-07-30T13:59:01.6303548Z   19 failing
2024-07-30T13:59:01.6303683Z 
2024-07-30T13:59:01.6303833Z   1) iModel
2024-07-30T13:59:01.6304021Z        ecefLocation for iModels:
2024-07-30T13:59:01.6304166Z 
2024-07-30T13:59:01.6304312Z       spatialToEcef
2024-07-30T13:59:01.6304668Z       + expected - actual
2024-07-30T13:59:01.6304805Z 
2024-07-30T13:59:01.6304978Z       -false
2024-07-30T13:59:01.6305154Z       +true
2024-07-30T13:59:01.6305323Z       
2024-07-30T13:59:01.6305546Z       at Context.<anonymous> (src\test\imodel\IModel.test.ts:1098:12)
2024-07-30T13:59:01.6305815Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6305976Z 
2024-07-30T13:59:01.6306112Z   2) iModel
2024-07-30T13:59:01.6306326Z        should be able to create a snapshot IModel and set geolocation by GCS:
2024-07-30T13:59:01.6306464Z 
2024-07-30T13:59:01.6306787Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6307019Z       + expected - actual
2024-07-30T13:59:01.6307126Z 
2024-07-30T13:59:01.6307343Z       -false
2024-07-30T13:59:01.6307529Z       +true
2024-07-30T13:59:01.6307708Z       
2024-07-30T13:59:01.6307934Z       at Context.<anonymous> (src\test\imodel\IModel.test.ts:1706:12)
2024-07-30T13:59:01.6308208Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6308373Z 
2024-07-30T13:59:01.6308518Z   3) iModel
2024-07-30T13:59:01.6308751Z        should be able to reproject with iModel coordinates to or from any other GeographicCRS:
2024-07-30T13:59:01.6308935Z 
2024-07-30T13:59:01.6309112Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6309623Z       + expected - actual
2024-07-30T13:59:01.6309774Z 
2024-07-30T13:59:01.6309962Z       -false
2024-07-30T13:59:01.6310150Z       +true
2024-07-30T13:59:01.6310326Z       
2024-07-30T13:59:01.6310543Z       at convertTest (src\test\imodel\IModel.test.ts:1766:65)
2024-07-30T13:59:01.6310828Z       at async Context.<anonymous> (src\test\imodel\IModel.test.ts:1876:7)
2024-07-30T13:59:01.6311020Z 
2024-07-30T13:59:01.6311166Z   4) iModel
2024-07-30T13:59:01.6311374Z        presence of a GCS imposes the ecef value:
2024-07-30T13:59:01.6311521Z 
2024-07-30T13:59:01.6311687Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6311950Z       + expected - actual
2024-07-30T13:59:01.6312270Z 
2024-07-30T13:59:01.6312468Z       -false
2024-07-30T13:59:01.6312658Z       +true
2024-07-30T13:59:01.6312839Z       
2024-07-30T13:59:01.6313084Z       at Context.<anonymous> (src\test\imodel\IModel.test.ts:2166:12)
2024-07-30T13:59:01.6313365Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6313537Z 
2024-07-30T13:59:01.6313693Z   5) GeoServices
2024-07-30T13:59:01.6314015Z        Interpret to completion an incomplete GeographicCRS
2024-07-30T13:59:01.6314259Z          should be able to interpret to completion britishNationalGridOld:
2024-07-30T13:59:01.6314416Z 
2024-07-30T13:59:01.6314574Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6314827Z       + expected - actual
2024-07-30T13:59:01.6314964Z 
2024-07-30T13:59:01.6315139Z       -false
2024-07-30T13:59:01.6315313Z       +true
2024-07-30T13:59:01.6315483Z       
2024-07-30T13:59:01.6315699Z       at completionTest (src\test\misc\GeoServices.test.ts:27:12)
2024-07-30T13:59:01.6315972Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:145:13)
2024-07-30T13:59:01.6316236Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6316397Z 
2024-07-30T13:59:01.6316533Z   6) GeoServices
2024-07-30T13:59:01.6316741Z        Interpret to completion an incomplete GeographicCRS
2024-07-30T13:59:01.6316975Z          should be able to interpret to completion EWRGCS:
2024-07-30T13:59:01.6317127Z 
2024-07-30T13:59:01.6317288Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6317540Z       + expected - actual
2024-07-30T13:59:01.6317677Z 
2024-07-30T13:59:01.6317856Z       -false
2024-07-30T13:59:01.6318040Z       +true
2024-07-30T13:59:01.6318217Z       
2024-07-30T13:59:01.6318429Z       at completionTest (src\test\misc\GeoServices.test.ts:27:12)
2024-07-30T13:59:01.6318706Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:231:13)
2024-07-30T13:59:01.6318972Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6319142Z 
2024-07-30T13:59:01.6319288Z   7) GeoServices
2024-07-30T13:59:01.6319497Z        Interpret to completion an incomplete GeographicCRS
2024-07-30T13:59:01.6319725Z          should be able to interpret to completion UTM27Z10:
2024-07-30T13:59:01.6319874Z 
2024-07-30T13:59:01.6320033Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6320284Z       + expected - actual
2024-07-30T13:59:01.6320418Z 
2024-07-30T13:59:01.6320594Z       -false
2024-07-30T13:59:01.6320772Z       +true
2024-07-30T13:59:01.6320942Z       
2024-07-30T13:59:01.6321157Z       at completionTest (src\test\misc\GeoServices.test.ts:27:12)
2024-07-30T13:59:01.6321433Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:332:13)
2024-07-30T13:59:01.6321699Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6321855Z 
2024-07-30T13:59:01.6322000Z   8) GeoServices
2024-07-30T13:59:01.6322209Z        Interpret to completion an incomplete GeographicCRS
2024-07-30T13:59:01.6322442Z          should be able to interpret to completion UTM27Z10B:
2024-07-30T13:59:01.6322590Z 
2024-07-30T13:59:01.6322752Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6323006Z       + expected - actual
2024-07-30T13:59:01.6323327Z 
2024-07-30T13:59:01.6323508Z       -false
2024-07-30T13:59:01.6323690Z       +true
2024-07-30T13:59:01.6323860Z       
2024-07-30T13:59:01.6324082Z       at completionTest (src\test\misc\GeoServices.test.ts:27:12)
2024-07-30T13:59:01.6324351Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:439:13)
2024-07-30T13:59:01.6324618Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6324776Z 
2024-07-30T13:59:01.6324923Z   9) GeoServices
2024-07-30T13:59:01.6325125Z        Interpretation of WKT GeographicCRS
2024-07-30T13:59:01.6325353Z          should be able to interpret airportGrid2007 WKT:
2024-07-30T13:59:01.6325508Z 
2024-07-30T13:59:01.6325664Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6326088Z       + expected - actual
2024-07-30T13:59:01.6326229Z 
2024-07-30T13:59:01.6326405Z       -false
2024-07-30T13:59:01.6326585Z       +true
2024-07-30T13:59:01.6326756Z       
2024-07-30T13:59:01.6326979Z       at interpretWKTTest (src\test\misc\GeoServices.test.ts:449:14)
2024-07-30T13:59:01.6327257Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:545:13)
2024-07-30T13:59:01.6327518Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6327678Z 
2024-07-30T13:59:01.6327823Z   10) GeoServices
2024-07-30T13:59:01.6328029Z        Interpretation of WKT GeographicCRS
2024-07-30T13:59:01.6328243Z          should be able to interpret denmarkED50 WKT:
2024-07-30T13:59:01.6328396Z 
2024-07-30T13:59:01.6328562Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6328826Z       + expected - actual
2024-07-30T13:59:01.6328962Z 
2024-07-30T13:59:01.6329139Z       -false
2024-07-30T13:59:01.6329324Z       +true
2024-07-30T13:59:01.6329498Z       
2024-07-30T13:59:01.6329717Z       at interpretWKTTest (src\test\misc\GeoServices.test.ts:449:14)
2024-07-30T13:59:01.6329992Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:619:13)
2024-07-30T13:59:01.6330223Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6330349Z 
2024-07-30T13:59:01.6330653Z   11) GeoServices
2024-07-30T13:59:01.6330829Z        Interpretation of WKT GeographicCRS
2024-07-30T13:59:01.6331025Z          should be able to interpret californiaStateZone2 WKT:
2024-07-30T13:59:01.6331153Z 
2024-07-30T13:59:01.6331320Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6331556Z       + expected - actual
2024-07-30T13:59:01.6331774Z 
2024-07-30T13:59:01.6331969Z       -false
2024-07-30T13:59:01.6332165Z       +true
2024-07-30T13:59:01.6332348Z       
2024-07-30T13:59:01.6332551Z       at interpretWKTTest (src\test\misc\GeoServices.test.ts:449:14)
2024-07-30T13:59:01.6332845Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:667:13)
2024-07-30T13:59:01.6333118Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6333283Z 
2024-07-30T13:59:01.6333431Z   12) GeoServices
2024-07-30T13:59:01.6333644Z        Interpretation of WKT GeographicCRS
2024-07-30T13:59:01.6333869Z          should be able to interpret utm84Zone34S WKT:
2024-07-30T13:59:01.6334017Z 
2024-07-30T13:59:01.6334189Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6334453Z       + expected - actual
2024-07-30T13:59:01.6334600Z 
2024-07-30T13:59:01.6334790Z       -false
2024-07-30T13:59:01.6334944Z       +true
2024-07-30T13:59:01.6335096Z       
2024-07-30T13:59:01.6335326Z       at interpretWKTTest (src\test\misc\GeoServices.test.ts:449:14)
2024-07-30T13:59:01.6335626Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:714:13)
2024-07-30T13:59:01.6335887Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6336033Z 
2024-07-30T13:59:01.6336184Z   13) GeoServices
2024-07-30T13:59:01.6336385Z        Interpretation of WKT GeographicCRS
2024-07-30T13:59:01.6336589Z          should be able to interpret utm84Zone32NGeoid WKT:
2024-07-30T13:59:01.6336736Z 
2024-07-30T13:59:01.6336902Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6337381Z       + expected - actual
2024-07-30T13:59:01.6337523Z 
2024-07-30T13:59:01.6337709Z       -false
2024-07-30T13:59:01.6337897Z       +true
2024-07-30T13:59:01.6338077Z       
2024-07-30T13:59:01.6338309Z       at interpretWKTTest (src\test\misc\GeoServices.test.ts:449:14)
2024-07-30T13:59:01.6338589Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:761:13)
2024-07-30T13:59:01.6338867Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6339038Z 
2024-07-30T13:59:01.6339189Z   14) GeoServices
2024-07-30T13:59:01.6339399Z        Interpretation of WKT GeographicCRS
2024-07-30T13:59:01.6339631Z          should be able to interpret utm84Zone18NGeoid WKT:
2024-07-30T13:59:01.6339952Z 
2024-07-30T13:59:01.6340120Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6340388Z       + expected - actual
2024-07-30T13:59:01.6340525Z 
2024-07-30T13:59:01.6340711Z       -false
2024-07-30T13:59:01.6340901Z       +true
2024-07-30T13:59:01.6341079Z       
2024-07-30T13:59:01.6341307Z       at interpretWKTTest (src\test\misc\GeoServices.test.ts:449:14)
2024-07-30T13:59:01.6341591Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:804:13)
2024-07-30T13:59:01.6341865Z       at processImmediate (node:internal/timers:483:21)
2024-07-30T13:59:01.6342027Z 
2024-07-30T13:59:01.6342177Z   15) GeoServices
2024-07-30T13:59:01.6342374Z        Verify list of CRS
2024-07-30T13:59:01.6342575Z          should get all CRS:
2024-07-30T13:59:01.6342712Z 
2024-07-30T13:59:01.6342883Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6343149Z       + expected - actual
2024-07-30T13:59:01.6343291Z 
2024-07-30T13:59:01.6343479Z       -false
2024-07-30T13:59:01.6343671Z       +true
2024-07-30T13:59:01.6343854Z       
2024-07-30T13:59:01.6344082Z       at validateCRSList (src\test\misc\GeoServices.test.ts:831:14)
2024-07-30T13:59:01.6344372Z       at async Context.<anonymous> (src\test\misc\GeoServices.test.ts:843:7)
2024-07-30T13:59:01.6344557Z 
2024-07-30T13:59:01.6344707Z   16) GeoServices
2024-07-30T13:59:01.6344901Z        Verify list of CRS
2024-07-30T13:59:01.6345124Z          should return CRS that are in the specified range (1):
2024-07-30T13:59:01.6345287Z 
2024-07-30T13:59:01.6345450Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6345722Z       + expected - actual
2024-07-30T13:59:01.6345844Z 
2024-07-30T13:59:01.6346029Z       -false
2024-07-30T13:59:01.6346197Z       +true
2024-07-30T13:59:01.6346382Z       
2024-07-30T13:59:01.6346586Z       at validateCRSList (src\test\misc\GeoServices.test.ts:831:14)
2024-07-30T13:59:01.6346852Z       at async Context.<anonymous> (src\test\misc\GeoServices.test.ts:848:7)
2024-07-30T13:59:01.6347053Z 
2024-07-30T13:59:01.6347206Z   17) GeoServices
2024-07-30T13:59:01.6347370Z        Verify list of CRS
2024-07-30T13:59:01.6347556Z          should return CRS that are in the specified range (2):
2024-07-30T13:59:01.6347713Z 
2024-07-30T13:59:01.6347876Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6348107Z       + expected - actual
2024-07-30T13:59:01.6348252Z 
2024-07-30T13:59:01.6348434Z       -false
2024-07-30T13:59:01.6348589Z       +true
2024-07-30T13:59:01.6348740Z       
2024-07-30T13:59:01.6348972Z       at validateCRSList (src\test\misc\GeoServices.test.ts:831:14)
2024-07-30T13:59:01.6349264Z       at async Context.<anonymous> (src\test\misc\GeoServices.test.ts:853:7)
2024-07-30T13:59:01.6349447Z 
2024-07-30T13:59:01.6349596Z   18) GeoServices
2024-07-30T13:59:01.6349792Z        Verify list of CRS
2024-07-30T13:59:01.6350011Z          should return CRS that are in the specified range (3):
2024-07-30T13:59:01.6350172Z 
2024-07-30T13:59:01.6350341Z       AssertionError: expected false to be true
2024-07-30T13:59:01.6350609Z       + expected - actual
2024-07-30T13:59:01.6350748Z 
2024-07-30T13:59:01.6350938Z       -false
2024-07-30T13:59:01.6351327Z       +true
2024-07-30T13:59:01.6351513Z       
2024-07-30T13:59:01.6351744Z       at validateCRSList (src\test\misc\GeoServices.test.ts:831:14)
2024-07-30T13:59:01.6352034Z       at async Context.<anonymous> (src\test\misc\GeoServices.test.ts:858:7)
2024-07-30T13:59:01.6352219Z 
2024-07-30T13:59:01.6352373Z   19) GeoServices
2024-07-30T13:59:01.6352573Z        Verify list of CRS
2024-07-30T13:59:01.6352820Z          should retrieve the whole list of CRS and validate the properties for a few selected CRS.:
2024-07-30T13:59:01.6353010Z 
2024-07-30T13:59:01.6353177Z       AssertionError: expected +0 to equal 4
2024-07-30T13:59:01.6353405Z       + expected - actual
2024-07-30T13:59:01.6353553Z 
2024-07-30T13:59:01.6353897Z       -0
2024-07-30T13:59:01.6354075Z       +4
2024-07-30T13:59:01.6354246Z       
2024-07-30T13:59:01.6354471Z       at Context.<anonymous> (src\test\misc\GeoServices.test.ts:892:14)
2024-07-30T13:59:01.6354657Z 
2024-07-30T13:59:01.6354745Z 
2024-07-30T13:59:01.6354866Z 
2024-07-30T13:59:01.6355319Z ##[warning]2 tests skipped in @itwin/core-backend
2024-07-30T13:59:01.6355798Z Returned error code: 19
2024-07-30T13:59:01.6356093Z "@itwin/core-backend" failed to build.
```

**Pull request merged**